plugins/reports/counter/classes/releases/CounterRelease3_0.inc.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file plugins/generic/sushiLite/classes/releases/CounterRelease3_0.inc.php
+ *
+ * Copyright (c) 2014 University of Pittsburgh
+ * Distributed under the GNU GPL v2 or later. For full terms see the file docs/COPYING.
+ *
+ * @class CounterRelease3_0
+ * @ingroup plugins_reports_counter
+ *
+ * @brief A COUNTER report, release 3.0
+ */
+
+import('plugins.reports.counter.classes.releases.CounterRelease');
+
+class CounterRelease3_0 extends CounterRelease {
```

Comment:
Yes, I like this, though I think I should get the potential for composition ready now rather than later.

JR1, AR1, etc. is a
CounterReport
  has a CounterRelease